app/controllers/govspeak_controller.rb

```diff
@@ -0,0 +1,10 @@
+class GovspeakController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  require "govspeak"
+
+  def show
+    doc = Govspeak::Document.new params[:input]
+    render json: { html: doc.to_html }
+  end
+end
```

Comment:
Can you stop the markdown param appearing in the logs ?

Because it will make it hard to see anything else

This shows how

https://stackoverflow.com/questions/63147598/disable-certain-parameters-from-being-logged-in-ruby-on-rails

Reply:
ive added     Rails.logger.silence  to that method